### PR TITLE
[IMP] website: introduce `s_empowerment` snippet

### PR DESCRIPTION
**Pull request by chgo-odoo**
This PR introduces the new `s_empowerment` snippet.

- requires https://github.com/odoo/design-themes/pull/875

image

task-4077615
Part of task-4077427

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website/views/snippets/snippets.xml [hunk 1]
```diff
@@ -677,7 +680,7 @@
     </div>
 
     <!--  Vertical Alignment -->
-    <div data-js="vAlignment" id="row_valign_snippet_option" data-selector=".s_text_image, .s_image_text, .s_three_columns, .s_showcase, .s_numbers, .s_faq_collapse, .s_references, .s_accordion_image" data-target=".row">
+    <div data-js="vAlignment" id="row_valign_snippet_option" data-selector=".s_text_image, .s_image_text, .s_three_columns, .s_showcase, .s_numbers, .s_faq_collapse, .s_references, .s_accordion_image, .s_empowerment" data-target=".row">
```
Comment (xlu-odoo) [inline addons/website/views/snippets/snippets.xml:683]: If you do remove the ability to pass in col-mode, then remove this change please : )

**Review by xlu-odoo — CHANGES_REQUESTED** [review]
Good work! 🚀 
A few remarks below

**Review by xlu-odoo — APPROVED** [review]
LGTM 🚀

File: addons/website/views/snippets/s_empowerment.xml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+<template id="s_empowerment" name="Connections">
+    <section class="s_empowerment pt24 pb24">
+        <div class="container-fluid">
+            <div class="row o_grid_mode" data-row-count="12">
+                <div class="o_grid_item g-height-12 g-col-lg-7 col-lg-7 justify-content-center" style="grid-area: 1 / 1 / 13 / 8; --grid-item-padding-x: 24px;">
+                    <span class="s_cta_badge d-inline-block my-3 border rounded py-2 px-3 o_cc o_cc1 o_animable" data-snippet="s_cta_badge" data-name="CTA Badge" style="border-radius: 32px !important;">
+                        <i class="fa fa-fw fa-info-circle o_not-animable" role="img"/>&#160;&#160;What do you want to promote&amp;nbsp;? &#160;&#160;&#160;&#160;<a href="#">See more <i class="fa fa-long-arrow-right" role="img"/></a>
+                    </span>
+                    <h1 class="display-5">Empowering Your Success<br/>with Every Solution.</h1>
```
Comment (bso-odoo) [inline addons/website/views/snippets/s_empowerment.xml:12]: I think display only goes up to 4 on `h1`.

**Review by bvr-odoo — APPROVED** [review]

**Comment by bvr-odoo** [comment]
Commit(s) moved into

- https://github.com/odoo/odoo/pull/180736

**Pull request closed without merging**